src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -518,7 +510,7 @@ namespace winrt::TerminalApp::implementation
 
         // Add a event handlers to the new panes' GotFocus event. When the pane
         // gains focus, we'll mark it as the new active pane.
-        _AttachEventHandlersToControl(newPane->Id().value(), control);
+        _AttachEventHandlersToControl(newPane->Id().value(), newPane->GetTerminalControl());

```

Comment:
this will break if you have the debug pane made since newPane will be a parent pane and have no control. Similarly handlers won't be added properly to the two children. This could be fixed like this:

https://github.com/Rosefield/terminal/commit/84beb7afdad8c09ef0fbc9f69be1b857a507a015#diff-83d260047bed34d3d9d5a12ac62008b65bd6dc5f3b9642905a007c3efce27efdR498

Reply:
Great catch, thanks!